tools/apidoc/src/log.ts

```diff
@@ -1,9 +0,0 @@
-//
-// Copyright 2024 DXOS.org
-//
-
-import chalk from 'chalk';
-
-export const info = (...args: any[]) => console.warn(chalk.cyan('[info]'), ...args);
-export const warn = (...args: any[]) => console.warn(chalk.yellow('[warn]'), ...args);
-export const err = (...args: any[]) => console.warn(chalk.red('[err]'), ...args);
```

Comment:
This is perfectly correct generally useful code. I chose to remove it since it's now unused, but would be happy to leave it as well.

Reply:
I'm wrong, it's being used elsewhere. I'll add it back.